evansmusomi/flask-wallet
# Incorporates Challenge 1 Feedback

### What's this PR do?

Incorporates feedback on Challenge 1 code quality and improves README file

### Where should the reviewer start?

ui/css/style.css

### How should this be manually tested?

View the README file and any of the HTML files to verify that the CSS changes are valid

### Any background context you want to provide?

Used Code Climate and Codacy to identify areas for improvement.
